frontend/lib/Repository/AdminRepository/users_list_repository.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:adpv_frontend/Common/urls.dart';
+import 'package:adpv_frontend/Repository/EndpointRepository/endpoint_rest_repository.dart';
+import 'package:adpv_frontend/Repository/UserRepository/auth_gateway.dart';
+import 'package:dio/dio.dart';
+
+import '../UserRepository/user_gateway.dart';
+
+class UserGroups {
+  int id;
+  String name;
+
+  UserGroups(this.id, this.name);
+
+  UserGroups.fromJson(Map json)
+      : id = json["id"],
+        name = json["name"];
+}
+
+class UserListData {
```

Comment:
Nie uważam, żeby to była dobra nazwa. Sugeruje ona jakąś listę, natomiast tutaj mamy pojedynczego user'a. Dużo lepszym pomysłem, byłoby użycie po prostu `User`

Reply:
UserListData jest lepsza niż UserDataList, a samo User jest mylące bo już istnieje

Reply:
Hmmm, a co żeby użyć `UserData`? Albo tamtego user'a przemianować na `UserForAuthorization`? Na pewno pozbyłbym się `List` z nazwy